src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<Car> createdCars) {
+        cars = new ArrayList<>(createdCars);
+    }
+
+    public List<Car> moveCars(MoveStrategy moveStrategy) {
+        List<Car> carAfterRace = new ArrayList<>();
+        for (Car car : cars) {
+            if (moveStrategy.moveOrNot()) {
+                car.moveForward();
+            }
+            carAfterRace.add(new Car(car.getCarName(), car.getPosition()));
```

Comment:
getter를 사용하지 않고도 똑같은 동작을 하는 식을 만들어볼 수 있을 것 같아요! 🤗
```suggestion
                car = car.moveForward();
            }
            carAfterRace.add(car);
```